bucket/janet.json

```diff
@@ -0,0 +1,46 @@
+{
+    "homepage": "https://janet-lang.org",
+    "description": "Janet is a lisp-like, embeddable, functional and imperative programming language and bytecode interpreter.",
+    "license": "MIT",
+    "version": "1.13.1",
+    "architecture": {
+        "32bit": {
+            "hash": "627a0dccef7815fdde735caf3ea2eb3e16f2ec26ac3d0cc365cf8ea67589d5e3",
+            "url": "https://github.com/janet-lang/janet/releases/download/v1.13.1/janet-v1.13.1-windows-x86-installer.msi"
+        },
+        "64bit": {
+            "hash": "ef5d32257594c8ab54aaa52fe8e84bb951a4883bbb6f6848e845759122d33a35",
+            "url": "https://github.com/janet-lang/janet/releases/download/v1.13.1/janet-v1.13.1-windows-x64-installer.msi"
+        }

```

Comment:
64bit and then 32bit
url first, hash after

Reply:
Done